### PR TITLE
plugins(acme) add document for external account binding

### PR DESCRIPTION
**Pull request by fffonion**

### CODE REVIEW

File: app/_hub/kong-inc/acme/index.md [hunk 1]
```diff
@@ -79,6 +79,14 @@ params:
       default: "`false`"
       description: |
         If you are using Let's Encrypt, you must set this to true to agree the [Terms of Service](https://letsencrypt.org/repository/).
+    - name: eab_kid
+      required: false
+      description: |
+        External account binding (EAB) key id. User usually doesn't need to set this unless explictly required by CA.
```
Comment (tlblessing) [inline app/_hub/kong-inc/acme/index.md:85]:
```suggestion
        External account binding (EAB) key id. You usually don't need to set this unless it is explicitly required by the CA.
```

File: app/_hub/kong-inc/acme/index.md [hunk 2]
```diff
@@ -79,6 +79,14 @@ params:
...
+    - name: eab_hmac_key
+      required: false
+      description: |
+        External account binding (EAB) base64 encoded URL string of the HMAC key. User usually doesn't need to set this unless explictly required by CA.
```
Comment (tlblessing) [inline app/_hub/kong-inc/acme/index.md:89]:
```suggestion
        External account binding (EAB) base64-encoded URL string of the HMAC key. You usually don't need to set this unless it is explicitly required by the CA.
```

File: app/_hub/kong-inc/acme/index.md [hunk 3]
```diff
@@ -116,6 +124,14 @@ params:
 
     To configure storage type other than `kong`, please refer to [lua-resty-acme](https://github.com/fffonion/lua-resty-acme#storage-adapters).
 
+
+Externel account binding (EAB) is supported as long as `eab_kid` and `eab_hmac_key` is provided.
+
+The following CA provider's external account can be registered automatically and user doesn't need
```
Comment (tlblessing) [inline app/_hub/kong-inc/acme/index.md:130]:
```suggestion
The following CA provider's external account can be registered automatically, without specifying
```

File: app/_hub/kong-inc/acme/index.md [hunk 4]
```diff
@@ -116,6 +124,14 @@ params:
...
+to specify `eab_kid` or `eab_hmac_key`:
```
Comment (tlblessing) [inline app/_hub/kong-inc/acme/index.md:131]:
```suggestion
the `eab_kid` or `eab_hmac_key`:
```

File: app/_hub/kong-inc/acme/index.md [hunk 5]
```diff
@@ -116,6 +124,14 @@ params:
 
     To configure storage type other than `kong`, please refer to [lua-resty-acme](https://github.com/fffonion/lua-resty-acme#storage-adapters).
 
+
+Externel account binding (EAB) is supported as long as `eab_kid` and `eab_hmac_key` is provided.
```
Comment (tlblessing) [inline app/_hub/kong-inc/acme/index.md:128]: Not sure if this is both or either (or)?

```suggestion
Externel account binding (EAB) is supported as long as `eab_kid` and `eab_hmac_key` are provided.
```

**Review by tlblessing — CHANGES_REQUESTED** [review]
Thanks for the PR @fffonion. Suggested corrections. It looks like the versions file needs to be updated, as current docs only show 0.2.7. 

image

**Review by tlblessing — CHANGES_REQUESTED** [review]
We also need version 0.2.13 in versions file. The plugin is not displaying in the staged doc.

File: app/_data/extensions/kong-inc/acme/versions.yml [hunk 6]
```diff
@@ -1,2 +1,3 @@
+- release: 0.2.11
```
Comment (tlblessing) [inline app/_data/extensions/kong-inc/acme/versions.yml:1]:
```suggestion
- release: 0.2.13
- release: 0.2.11
```

**Review by tlblessing — CHANGES_REQUESTED** [review]

**Comment by tlblessing** [comment]
Need to troubleshoot why the plugin won't appear in the staging hub. Already added in versions.

**Comment by tlblessing** [comment]
Tested all versions from hub, docs now display ACME https://deploy-preview-2491--kongdocs.netlify.app/hub/kong-inc/acme/

**Review by tlblessing — APPROVED** [review]
@fffonion please review my edits, and if all looks good to you, merge the PR.

**Comment by tlblessing** [comment]
Merging so this doesn't sit.

**Comment by fffonion** [comment]
I missed the notification, thanks @tlblessing !

**Pull request merged**